src/throughputmanagement.c

```diff
@@ -63,7 +63,15 @@ void run_ntttcp_throughput_management(struct ntttcp_test_endpoint *tep)
 		req.tv_sec = tep->test->warmup;
 		
 		while(nanosleep(&req, &rem) == -1){
-			req = rem;
+			if (errno == EINTR) {
+				req = rem;
+			} else if (errno == EFAULT) {
+				PRINT_ERR("EFAULT: Problem with copying information from user space.");
+				break;
+			} else if (errno == EINVAL) {
+				PRINT_ERR("EINVAL: The time specified to sleep was not in the range [0,999999999]");
+				break;
+			}
```

Comment:
Use perror or strerror to convert any unexpected error code into a message. Don't continue to run the program on unexpected errors.

Reply:
Makes sense. Updated the code.

Reply:
Remove the special handling of EFAULT and EINVAL. Quit the program on unexpected errors.

Reply:
Thank you, I updated the code.

Reply:
It looks good now, I'm happy!